views/templates/PublishToRemote/index.tpl

```diff
@@ -0,0 +1,40 @@
+<?php
+use oat\tao\helpers\Template;
+?>
+
+
+<div class="main-container flex-container-main-form">
+    <h2><?=__('Publish "%s"', _dh(get_data('delivery-label')))?></h2>
+    <div id="form-container" class="form-content">
+        <div class="xhtml_form">
+            <form action="<?= get_data('submit-url'); ?>">
+                <input type="hidden" value="<?= get_data('delivery-uri') ?>" name="delivery-uri">
+                <?php if (has_data('warning')) :?>
+                <div class='feedback-warning'>
+                    <?= get_data('warning')?>
+                </div>
+                <?php endif;?>
+
+                <?php if (count(get_data('remote-environments')) > 0) :?>
+                    <h3><?= __('Remote environment(s)')?></h3>
+                    <ul class="plain">
+                        <?php foreach (get_data('remote-environments') as $index => $environment) :?>
+                            <li>
+                                <input type="checkbox" name="remote-environments[]" value="<?= $environment->getUri();?>" id="remote-environment-<?= $index ?>">
```

Comment:
Branch with this feature (disabling remote environments) is not merged to develop yet, therefore we cannot use it yet. We will add this check in the PR where it will be merged into develop.